UWP/PPSSPP_UWPMain.cpp

```diff
@@ -502,7 +502,7 @@ bool System_MakeRequest(SystemRequestType type, int requestId, const std::string
 		std::vector<std::string> supportedExtensions = {};
 		switch ((BrowseFileType)param3) {
 		case BrowseFileType::BOOTABLE:
-			supportedExtensions = { ".cso", ".bin", ".iso", ".elf", ".pbp", ".zip" };
+			supportedExtensions = { ".chd", ".cso", ".bin", ".iso", ".elf", ".pbp", ".zip" };
```

Comment:
just wondering why `.prx` extension can't be found in this list :) since prx files can be run on PPSSPP too (ie. sample files from PSPSDK)

Reply:
was probably just forgotten when the list was created. I'll add it.